src/app/courses/courses.module.ts

```diff
@@ -1,18 +1,21 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { CoursesListComponent } from './courses-list/courses-list.component';
+import { FormsModule } from '@angular/forms';
 import { CourseItemComponent } from './course-item/course-item.component';
 import { FilterFormComponent } from './filter-form/filter-form.component';
 import { CoursesPageComponent } from './courses-page/courses-page.component';
 import { SharedModule } from '../shared/shared.module';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 
 
 
 @NgModule({
-  declarations: [CoursesListComponent, CourseItemComponent, FilterFormComponent, CoursesPageComponent],
+  declarations: [CourseItemComponent, FilterFormComponent, CoursesPageComponent],
   imports: [
     CommonModule,
     SharedModule,
+    FontAwesomeModule,
+    FormsModule
```

Comment:
Consider keeping `FontAwesomeModule` and `FormsModule` as a part of the `SharedModule`. So that we could use them across application by `SharedModule` importing.